src/displayapp/icons/music/disc.cpp

```diff
@@ -20,91 +20,40 @@
 #include "lvgl/lvgl.h"
 
 #ifndef LV_ATTRIBUTE_MEM_ALIGN
-#define LV_ATTRIBUTE_MEM_ALIGN
+  #define LV_ATTRIBUTE_MEM_ALIGN
 #endif
 
 #ifndef LV_ATTRIBUTE_IMG_DISC
-#define LV_ATTRIBUTE_IMG_DISC
+  #define LV_ATTRIBUTE_IMG_DISC
 #endif
 
 const LV_ATTRIBUTE_MEM_ALIGN LV_ATTRIBUTE_IMG_DISC uint8_t disc_map[] = {
-    0xbd, 0xc1, 0xbe, 0xff,  /* Color of index 0: foreground */
```

Comment:
These disc files are generated by LVGL image converter. I think they should be .c files and not formatted. Can you rename them instead, so they get ignored?